### PR TITLE
Updated broken links for Applying Patches section

### PR DESCRIPTION
**Pull request by asrar7787**
Updated broken links for Applying Patches section

## Purpose of this pull request

This pull request fixes broken links for Applying Patches section.

## Affected pages

- https://experienceleague.adobe.com/docs/commerce-operations/upgrade-guide/patches/overview.html?lang=en#applying-patches

### CODE REVIEW

**Review by jeff-matthews — COMMENTED** [review]
@asrar7787, the links work ok for me. Are you getting a 404 when you click them?

**Comment by asrar7787** [comment]
Yes, it’s showing 404 for me.

**Comment by jeff-matthews** [comment]
Strange... what web browser are you using?

**Comment by asrar7787** [comment]
chrome, safari

File: help/upgrade/patches/overview.md
Comment (jeff-matthews) [inline help/upgrade/patches/overview.md]:
```suggestion
-  [Command line](/help/upgrade/patches/apply.md#command-line)
```

File: help/upgrade/patches/overview.md
Comment (jeff-matthews) [inline help/upgrade/patches/overview.md]:
```suggestion
-  [Composer](/help/upgrade/patches/apply.md#composer)
```

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
Let's try something a little different based on the [link syntax guidance](https://experienceleague.adobe.com/docs/authoring-guide-exl/using/markdown/linking.html?lang=en#root-links) in the authoring guide.

**Comment by jeff-matthews** [comment]
I checked with some colleagues internally and this doesn't appear to be a problem for anyone else and that the current syntax is valid. However, I am still interested in why you're getting 404s @asrar7787. 

When I check the code on the page, everything looks correct:
Screen Shot 2022-02-24 at 3 07 11 PM

**Comment by asrar7787** [comment]
I would not exactly know why I was getting 404, I was testing the 404
behaviour few times before opening the PR, if this is a red herring, happy
for you to close the PR.

Thanks

**Comment by jeff-matthews** [comment]
Let's try using the documented link syntax and see if that helps. We use the current syntax in several places so I'd like to know if this is a problem for others.

File: help/upgrade/patches/overview.md
Comment (asrar7787) [reply help/upgrade/patches/overview.md]: Committed your suggestions 👍 and thanks for sharing the link syntax guidelines. The idea of setting up the docs repo on local development machine and maintain the doc is great. But as I don't actively update/maintain the docs repo, and it might be the case for most of us, I tend to go for 'Converting URLs to hyperlinks' route but need to follow the syntax.

Comment (jeff-matthews) [reply help/upgrade/patches/overview.md]: No problem. After we merge this, please check if links work for you and report back here.

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**
